packages/@react-aria/table/src/useTableColumnResize.ts

```diff
@@ -79,9 +82,12 @@ export function useTableColumnResize<T>(props: AriaTableColumnResizeProps<T>, st
         }
       }
     },
-    onMoveEnd() {
+    onMoveEnd({pointerType}) {
       columnResizeWidthRef.current = 0;
       document.body.style.cursor = cursor.current;
+      if (pointerType === 'mouse') {
+        stateRef.current.onColumnResizeEnd(item);
```

Comment:
I noticed that onColumnResizeEnd is being fired twice now per mouse resize in https://reactspectrum.blob.core.windows.net/reactspectrum/64ad9faffb499da09831a4beeb35e78613d787a2/storybook/index.html?path=/story/tableview--allowsresizing-oncolumnresizeend-action&providerSwitcher-scale=large&providerSwitcher-toastPosition=bottom